api/tests/app/test_ml_models.py

```diff
@@ -0,0 +1,397 @@
+import shutil
+
+import pytest
+import os
+import json
+
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions.
+# This is why the IDE cannot recognize them.
+from api.tests.utils.fixtures_tests import config_rollback_cameras, rollback_camera_config
+
+
+def pascal_case_to_snake_case(parameters):
```

Comment:
Is this function not the same as the one defined in the app? Why not define it in utils and import in both?

Reply:
But if it is the same implementation I see no value in copying it instead of importing it, duplication of code is even more prone to bugs, right?

Reply:
If we want to be protected against these bugs an unitary test of  the method pascal_case_to_snake_case should be implemented, but I don't see that much value in that

Reply:
Okey, no problem I will import that function.

Reply:
Actually, I realize that when I create the function, it was because I need the same functionality as pascal_case_to_snake_case defined in ml_models.py, but instead of giving an Object as an argument, it is necessary to provide a dictation. The function defined in test_ml_models.py differs in the for statement, the iterator is different.

Reply:
So we need to keep the function